region/lowernorfair/east.json

```diff
@@ -1602,7 +1610,19 @@
           "to": [
             {
               "id": 1,
-              "requires": null,
+              "requires": [
+                {"or": [
+                  "heatProof",
+                  "canHeatRun"
+                ]},
+                {"cost": [
+                  {"heatFrames": 450},
+                  {"enemyDamage": [{
```

Comment:
I feel like that enemy damage probably shouldn't be mandatory if you're heatproof?

Reply:
Will fix